packages/ui/src/bounty/components/BountiesList.tsx

```diff
@@ -0,0 +1,59 @@
+import React, { memo } from 'react'
+import styled from 'styled-components'
+
+import { BountyOrderByInput } from '@/common/api/queries'
+import { CountBadge } from '@/common/components/CountBadge'
+import { Loading } from '@/common/components/Loading'
+import { Pagination } from '@/common/components/Pagination'
+import { SimpleSelect } from '@/common/components/selects'
+import { TextBig } from '@/common/components/typography'
+import { NotFoundText } from '@/common/components/typography/NotFoundText'
+import { useSort } from '@/common/hooks/useSort'
+
+import { useBounties } from '../hooks/useBounties'
+
+import { BountyListItem } from './BountyListItem/BountyListItem'
+
+export const BountiesList = memo(() => {
+  const { order, getSortProps } = useSort<BountyOrderByInput>('createdAt')
+  const { onSort, isDescending } = getSortProps('createdAt')
+  const { isLoading, bounties, pagination } = useBounties({ order })
+
+  if (isLoading) {
+    return <Loading />
+  }
+
+  if (!bounties?.length) {
+    return <NotFoundText>No Bounties</NotFoundText>
+  }
+
+  return (
+    <div>
+      <Title bold value>
+        Bounties
+        <Counter count={bounties.length} />
+      </Title>
+      <SimpleSelect
+        title="Sorting"
+        options={['Latest', 'Earliest']}
```

Comment:
Its better to extract this to `const` out of component so it won't trigger extra re-render